guide/designing-products/user-research-resources.md

```diff
@@ -37,29 +37,53 @@ Below is a collection of published user research findings around Bitcoin, from a
 ## Research papers
 
 ###### [Non-adoption of crypto-assets: Exploring the role of trust, self-efficacy, and risk](https://voskart.de/pdf/crypto_adoption.pdf)
+
+*Over the last years, crypto-assets have gained significant interest from private investors, academia, and industry. While the user population and their motivations, perceptions, and behaviors have been studied, non-adopters and factors influencing their decision have been left unexplored. This work fills this knowledge gap and sheds light on the effects of trust, perceived self-efficacy, and risk, which have been shown to be the key antecedents to technology acceptance, on the adoption intention of non-users.*
```

Comment:
```suggestion
A survey of 404 users and non-users of Bitcoin about the roles of trust, self-efficacy, and risk on adoption intention.
```
I know this is maybe not doing the full survey justice, but how about something really short and easy to parse? I personally have a hard time with the academic language.